dukan/src/main/kotlin/net/thechance/dukan/repository/DukanRepository.kt

```diff
@@ -153,9 +153,16 @@ interface DukanRepository : JpaRepository<Dukan, UUID> {
     ): Page<Dukan>
 
     @Modifying
-    @Query("UPDATE Dukan d SET d.status = :status WHERE d.id = :dukanId")
+    @Query("UPDATE Dukan d SET d.status = :activationStatus WHERE d.id = :dukanId")
```

Comment:
```suggestion
    @Query("UPDATE Dukan d SET d.status = :status WHERE d.id = :dukanId")
```
I think you modified it by mistake